tests/fixtures/factory.py

```diff
@@ -0,0 +1,28 @@
+
+import pytest
+
+from django.db.models import Max
+
+
+def _factory(**kwargs):
```

Comment:
I think this function deserves a little docstring. Mostly, the arguments are what I would love to see documented. :)